etc/profile-a-l/lz4.profile

```diff
@@ -0,0 +1,14 @@
+# Firejail profile for lz4
+# Description: Compress or decompress .lz4 files
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include lz4.local
+# Persistent global definitions
+include globals.local
+
+# mkinitcpio compression support
+private-etc mkinitcpio*
```

Comment:
```suggestion
```

Same as the following comment:

* <https://github.com/netblue30/firejail/pull/5656#pullrequestreview-1294491046>

Reply:
Fixed, thx!